metabase/metabase
# Add schedule fields and default engine to the formik database form

Epic https://github.com/metabase/metabase/issues/26178

How to test:
- `yarn storybook` -> `DatabaseForm`
- H2 -> Advanced options -> Choose when syncs and scans happen
- Make sure it is possible to change schedules and they are reflected in submit values

<img width="1475" alt="Screenshot 2022-12-01 at 14 56 29" src="https://user-images.githubusercontent.com/8542534/205058618-2e19c11b-4484-42e4-98be-ec881503f89c.png">
